src/content/en/2020/seo.md

```diff
@@ -23,3 +23,835 @@ queries: 07_SEO
+  image="seo-visible-words-per-page-rendered.png",
+  caption="Visible Words Per Page (rendered)",
+  description="Bar chart showing the number of visible words per page (in rendered response) per percentile (10, 25, 50, 75, and 90). The median mobile site displays 11.5% more words when rendered than its raw HTML counterpart.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=961460345&format=interactive",
+  sheets_gid="775602646",
+  sql_file="pages_wpt_bodies_by_device_and_percentile.sql"
+  )
+}}
+
+The median mobile site also displays 11.5% more words when rendered than its raw HTML counterpart.
+
+Across our sample set, there are disparities across the combination of mobile/desktop and rendered/non-rendered. This suggests that although search engines are continually improving in this area, most sites across the web are missing out on opportunities to improve their Organic Search visibility through a stronger focus on ensuring their content is available and indexable. This is also an issue of concern because the lion’s share of available SEO tools do not crawl in the above combination of contexts and automatically identify this as an issue.
+
+This year, the median desktop home page was found to have 402 words and the mobile 348 words. While [last year](../2019/seo#word-count), the median desktop home page had 346 words, and the median mobile home page had a slightly lower word count at 306 words. This represents 16.2% and 13.7% growth respectively.
+
+### Headings
+
+Heading elements (h1-h6) act as a mechanism to visually indicate structure in a page’s content. Although these HTML elements don’t carry the weight they used to in search rankings, they still act as a valuable way to structure pages and signal other elements in the SERPs like Featured Snippets or other extraction methods that align with [Google’s new passage indexing](https://www.blog.google/products/search/search-on/).
+
+{{ figure_markup(
+  image="seo-presence-of-h-tag.png",
+  caption="Presence of H Tag",
+  description="Bar chart showing the percent of pages with a heading tag (level 1,2,3 and 4). Over 60% of pages feature H1 elements in both the mobile and desktop contexts. These numbers hover around 60%+ through H2 and H3 and around 40% for H4.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=2103713054&format=interactive",
+  sheets_gid="337739550",
+  sql_file="pages_wpt_bodies_by_device.sqll"
+  )
+}}
+
+Over 60% of pages feature H1 elements in both the mobile and desktop contexts.
+
+These numbers hover around 60%+ through H2 and H3. The incidence rate of H4 elements is lower than 4% suggesting that the level of specificity is not required for most pages or the developers style other headings tags differently to support the visual structure of the content.
```

Comment:
@aleyda what do you think on the usage of "tags" throughout the document?

I think for `meta` elements this may possibly make sense, but for actual HTML elements (`<h1>`, `<h2>`) we should use the terms "elements" and similarly for attributes( e.g. `href` attributes, `alt` attributes). Maybe one for editing but would be keen to hear your thoughts as to what they are known as in the SEO world.

```suggestion
These numbers hover around 60%+ through H2 and H3. The incidence rate of H4 elements is lower than 4% suggesting that the level of specificity is not required for most pages or the developers style other headings elements differently to support the visual structure of the content.
```

Reply:
Thanks @bazzadp ! This had been already pointed out and had already changed a few mentions of the headings to be referred as "elements" and and alt as attributed. Unfortunately in the SEO world we might over (and mis) use the "tags" term to refer to many of these :D But by all means, let's be correct in the way we refer to them! I thought I had already caught them all... I'll take another look. Thanks again :)